airmozilla/manage/static/manage/js/picturegallery.js

```diff
@@ -240,6 +248,26 @@ app.controller('PictureGalleryController', ['$scope', '$http',
             });
         };
 
+        $scope.deleteAllPictures = function() {
+            var csrf = $('input[name="csrfmiddlewaretoken"]').val();
+            var url = $scope.url('manage:picture_delete_all');
+            var data = {
+                'csrfmiddlewaretoken': csrf,
+                'event': $scope.current_event
+            };
+            $http({
+                method: 'POST',
+                url: url,
+                headers: {'Content-Type': 'application/x-www-form-urlencoded'},
+                data: serializeObject(data)
+            }).success(function(data) {
+                $scope.pictures = [];
```

Comment:
You're right. That's not going to be enough. We'll need two things
1. Something like this:
```javascript
$scope.pictures = $scope.pictures.filter(function(picture) {
    return picture.event !== CURRENT_EVENT;
});
```

And when we change the [view function](https://github.com/anurag90x/airmozilla/blob/Bug%231117886/airmozilla/manage/views.py#L3621-L3648) to also include this `event` key. 

Does that make sense?